extensions/interactions/ImageClickInput/directives/image-click-input-validation.service.spec.ts

```diff
@@ -20,7 +20,7 @@ import { TestBed } from '@angular/core/testing';
 
 import { AnswerGroup, AnswerGroupObjectFactory } from
   'domain/exploration/AnswerGroupObjectFactory';
-import { IImageClickInputCustomizationArgs } from
+import { ImageClickInputCustomizationArgs } from
   'interactions/customization-args-defs';
 /* eslint-disable max-len*/
```

Comment:
This can be removed?

Reply:
nope. Theres a path below it more than 80 chars.